airflow/www/api/v0/endpoints.py

```diff
@@ -9,19 +9,22 @@
     url_for, Markup, Blueprint, redirect, jsonify, abort, request, app, send_file
     )
 
-apiv1 = Blueprint('apiv1', __name__)
+apiv0 = Blueprint('apiv0', __name__)
 
 dagbag = models.DagBag(os.path.expanduser(conf.get('core', 'DAGS_FOLDER')))
 
-@apiv1.route('/task_info/get/<string:dag_id>/<string:task_id>', methods=['GET'])
+@apiv0.route('/dags/<string:dag_id>/tasks/<string:task_id>', methods=['GET'])
 def task_info(dag_id, task_id):
   """Returns a JSON with a task's public instance variables. """
   if dag_id not in dagbag.dags:
-    return jsonify({'error': 'Dag {} not found'.format(dag_id)})
+    return jsonify({'http_status': 404,
```

Comment:
please do this

response = jsonify("the error")
response.status_code = 404

to have a proper http status code being returned